tests/MyGame/Example/NestedStruct.py

```diff
@@ -18,48 +18,60 @@ def Init(self, buf, pos):
         self._tab = flatbuffers.table.Table(buf, pos)
 
     # NestedStruct
-    def A(self): return [self._tab.Get(flatbuffers.number_types.Int32Flags, self._tab.Pos + flatbuffers.number_types.UOffsetTFlags.py_type(0 + i * 4)) for i in range(2)]
+    def A(self, j = None):
+        if j==None:
+            return [self._tab.Get(flatbuffers.number_types.Int32Flags, self._tab.Pos + flatbuffers.number_types.UOffsetTFlags.py_type(0 + i * 4)) for i in range(self.ALength())]
+        else:
+            return self._tab.Get(flatbuffers.number_types.Int32Flags, self._tab.Pos + flatbuffers.number_types.UOffsetTFlags.py_type(0 + j * 4))
```

Comment:
There is no error checking for out of bounds. Should this be added?

If so, doing it in another PR is fine.

Reply:
I can quickly add it in as it should be changing the else to an elif and then adding a return None for out of bounds

Reply:
Changed to produce None on out of bounds, potentially we can change this with out of bound exceptions but that might be better to be done in another PR and depends on if that is the programming style wanted or the more functional approach of None works just as well.